internal/controller/progressive/progressive.go

```diff
@@ -142,18 +142,14 @@ func ProcessResource(
 	}
 
 	// is there currently an "upgrading" child?
-	currentUpgradingChildDef, err := ctlrcommon.FindMostCurrentChildOfUpgradeState(ctx, rolloutObject, common.LabelValueUpgradeInProgress, nil, false, c)
```

Comment:
this gets the value from the cache if you pass it false, but from the live API if you pass it true